lib/web_ui/dev/test_platform.dart

```diff
@@ -126,15 +120,12 @@ class BrowserPlatform extends PlatformPlugin {
         //  * Assets that are part of the engine sources, such as Ahem.ttf
         .add(_packageUrlHandler)
 
-        // Serves CanvasKit assets.
-        .add(_canvasKitHandler)
-
         // Serves files from the web_ui/build/ directory at the root (/) URL path.
         //
         // Includes:
         //  * Precompiles .js files for tests
         //  * Sourcemaps
```

Comment:
Time to update this comment?

Reply:
Done. I'm not going to list anything. This kind of comments go stale quickly.